src/Microsoft.VisualStudio.Services.Agent/Blob/DedupManifestArtifactClientFactory.cs

```diff
@@ -78,6 +80,7 @@ private DedupManifestArtifactClientFactory()
             Action<string> traceOutput,
             VssConnection connection,
             int maxParallelism,
+            IDomainId domainId,
```

Comment:
>domainId [](http://example.com/codeflow?start=22&length=8)

since this is a public repo do we need to document what `domain` means or can we defer until we expose the options?

Reply:
I'll punt on this, since 'DedupManifestClient' falls under the same category(ADO specific library)